### ISSUE TITLE
A couple of new features, please

### ISSUE THREAD

**Issue by WatislavB** [EZ64cool/obs-hadowplay#18]
Can I suggest a couple of cool features that I need?

- Desktop notification when the replay buffer is launched (something like "Repaly Buffer enabled\n<game_name>");
- Ability to disable sorting of records into folders;
- Ability to add game name to recording file name;
- Control the notification volume from the plugin settings window (I know that you can control the overall OBS volume in the volume mixer, but I would like to be able to change the plugin volume separately).

Please at least think about it. 🙏

**Comment by EZ64cool** [comment]
Yeah those are all great suggestions I'll look at getting them in for a future version 😄

**Comment by ThatJRadGuy** [comment]
+1 for the game name to be in the filename. This plugin is fantastic and solved the dilemma i’ve had with recording gameplay, and adding the game name to the filename would be the icing on the cake for me! Thank you!

**Comment by EZ64cool** [comment]
> Ability to disable sorting of records into folders;

This has now been added in the latest release
